### PR TITLE
Update Scout ref docs to v0.17.0

### PR DESCRIPTION
**Pull request by cdupuis**
### Proposed changes

Updating Docker Scout CLI docs to latest released version v0.17.0.

### CODE REVIEW

**Comment by cdupuis** [comment]
@thaJeztah @ChrisChinchilla does either of you spot why this action failed? The preview deploy seems fine for me.

**Comment by thaJeztah** [comment]
Did scout add a new subcommand (scout_repo)?

**Comment by cdupuis** [comment]
> Did scout add a new subcommand (scout_repo)?

Yes, we are deprecating `entitlement` over `repo list|enable|disable` as per our prior conversation.

**Comment by ChrisChinchilla** [comment]
@cdupuis It's failing due to some apparent broken links, let me take a look.

**Comment by ChrisChinchilla** [comment]
@cdupuis Looks like you fixed it :)

**Comment by cdupuis** [comment]
Hmm, not in the TOC yet. 👀

**Comment by ChrisChinchilla** [comment]
@cdupuis Yeah that needs updating, give me a sec and will do

File: _data/toc.yaml [hunk 1]
```diff
@@ -595,6 +595,14 @@ reference:
         title: docker scout quickview
       - path: /engine/reference/commandline/scout_recommendations/
         title: docker scout recommendations
+      - path: /engine/reference/commandline/scout_repo/
```
Comment (ChrisChinchilla) [inline _data/toc.yaml:598]: @thaJeztah I can see some inconsistency in the docs where we sometimes just saw the top level command and other times all the sub commands. Any preference?

**Comment by cdupuis** [comment]
This looks good to go now.

**Pull request merged**